test/clojure/core_test/portability.cljc

```diff
@@ -5,10 +5,10 @@
         exists? (boolean (if cljs?
                            ((resolve 'cljs.analyzer.api/resolve) &env var-sym)
                            (resolve var-sym)))]
-    `(if ~exists?
-       (do
+    (if ~exists?
```

Comment:
Can't unquote here, if we're not in a syntax quote.

Reply:
Ah right! I keep missing that one in all 3 copies of these changes 😆